libc/config/baremetal/api.td

```diff
@@ -52,6 +52,15 @@ def IntTypesAPI : PublicAPI<"inttypes.h"> {
   let Types = ["imaxdiv_t"];
 }
 
+def StdIOAPI : PublicAPI<"stdio.h"> {
+  let Macros = [
+    SimpleMacroDef<"EOF", "-1">,
```

Comment:
Can you add this definition in `libc/include/llvm-libc-macros/stdio-macros.h` instead?

Reply:
Done